packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -153,6 +153,13 @@ export const SelectItem = styled( Ariakit.SelectItem )(
 export const SelectedItemCheck = styled( Ariakit.SelectItemCheck )`
 	display: flex;
 	align-items: center;
+
+	// Keep checkmark vertically aligned at the top. Since the item text has a
+	// 28px line height and the checkmark is 24px tall, a (28-24)/2 = 2px margin
+	// is applied to keep the correct alignment between text and checkmark.
```

Comment:
```suggestion
	// Keep the checkmark vertically aligned at the top. Since the item text has a
	// 28px line height and the checkmark is 24px tall, a (28-24)/2 = 2px margin
	// is applied to keep the correct alignment between the text and the checkmark.
```